src/engraving/libmscore/splitMeasure.cpp

```diff
@@ -145,6 +144,10 @@ void Score::splitMeasure(Segment* segment)
             ticks2.setNumerator(ticks2.numerator() * mult);
         }
     }
+    if (ticks1.denominator() > 128 || ticks2.denominator() > 128) {
+        MScore::setError(MsError::CANNOT_SPLIT_MEASURE_TOO_SHORT);
+        return;
+    }
```

Comment:
Difficult to test, because of #8895. It doesn't crash, it doesn't split the measure, but also doesn't give any error message